### ISSUE TITLE
Python Den Level 7 - insert explainer video

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/rapid-router#1740]
## Task Description
[Level 7](https://www.codeforlife.education/pythonden/7/) of the Python Den game is the first level to introduce variable-type blocks in Blockly.

We created a short explainer video on how these blocks work, which we need to insert into the intro popup of level 7.

## Acceptance Criteria
- [ ] Video inserted into instruction pop-up

## Good Places to Start
An example of where we've already embedded a video in an intro popup: https://github.com/ocadotechnology/rapid-router/blob/master/game/messages.py#L481

### Analytics Requirements
- [ ] Views on video recorded through YouTube

### Copywrite Requirements
None required.

Ask a member of the Code for Life team for the link to the YouTube video.

**Comment by bella054** [comment]
assign me please :)

**Comment by SKairinos** [comment]
@lauracumming , can you please share the link to the video on YouTube. I cannot find it for some reason

**Comment by lauracumming** [comment]
https://youtu.be/Lw-MMIlGU1Y

**Comment by lauracumming** [comment]
Video link should also be in hint

<img width="1108" alt="Image" src="https://github.com/user-attachments/assets/a6cb076e-e8a7-4680-8ffd-fb0b1b700feb" />

**Comment by SKairinos** [comment]
Are we happy with the size of the video?

We generally have responsiveness issues in the old site so tempted to not include fixing this in this task.

Level Description:

![Image](https://github.com/user-attachments/assets/c1e69124-d87c-4743-8f15-b60dd3a47cc6)

Level Hint:

![Image](https://github.com/user-attachments/assets/b271e11a-8271-4eb1-ace2-6d364aa68820)

**Comment by lauracumming** [comment]
Please address this by looking at responsive design.

**Comment by SKairinos** [comment]
Hello @bella054. We'd like you to fix the layout of the videos before we can accept this task as completed. Please make the following changes:

1. The correct aspect ratio of YouTube videos is 16 / 9. [There's a CSS property for this](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio). You'll need to update the aspect ratio in both the description and the hint.
2.  In the level hint, please put the video on the left and the text on the right, similar to how it is in the description.

**Comment by SKairinos** [comment]
![Image](https://github.com/user-attachments/assets/edfd40a8-fd2d-4d29-8139-4656b489f1f0)

**Comment by SKairinos** [comment]
![Image](https://github.com/user-attachments/assets/0a4ab46c-6ae8-4f98-8f4f-fdf05d0f76a6)

**Comment by lauracumming** [comment]
@faucomte97 Add are showing on the YouTube placeholder which are not suitable for children. Is there a way to prevent this?

**Comment by lauracumming** [comment]
Also, I have a question with regards to the format, did we not just want to replicate level 19 Rapid Router?

<img width="1048" alt="Image" src="https://github.com/user-attachments/assets/18b0195b-0c42-462e-acbe-0c2263e9583b" />

**Comment by faucomte97** [comment]
I had the same thought yesterday - yes, we probably do want to do that

**Comment by SKairinos** [comment]
[Should disable related videos](https://developers.google.com/youtube/player_parameters#rel)

**Comment by SKairinos** [comment]
LGTM. Another PR changes videos to GitBook
